packages/web-components/src/components/rux-clock/rux-clock.tsx

```diff
@@ -119,17 +131,66 @@ export class RuxClock {
         )
     }
 
+    private _handleDateIn() {
+        if (!this.dateIn!.includes('-')) {
+            this._validateDateInUnix(this.dateIn!)
+        }
+        this._time = this.dateIn!
+        if (!this._rawTime) this._rawTime = new Date(this.dateIn!)
+        if (this._validateDateIn(this._rawTime)) {
+            this._timer = window.setInterval(() => {
+                this._updateTime()
+            }, 1000)
+        } else {
+            console.warn(
```

Comment:
We don't have to do the console warn here, it already will produce a console error. I figured this just tells you exactly what threw the error, but I'm okay with getting rid of it